packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import './Rating.scss';
+import { IRatingProps, RatingSize } from './Rating.Props';
+import { BaseComponent } from '../../common/BaseComponent';
+import { getId } from '../../utilities/object';
+import { css } from '../../utilities/css';
+
+export interface IRatingState {
+  rating: number;
+  focusedRating: number;
+}
+
+export class Rating extends BaseComponent<IRatingProps, IRatingState> {
+  public static defaultProps: IRatingProps = {
+    min: 1,
+    max: 5
+  };
+
+  private _id: string;
+  private _labelId: string;
+
+  constructor(props: IRatingProps) {
+    super(props);
+
+    this.state = {
+      rating: this._getInitialValue(props),
+      focusedRating: null
+    };
+
+    this._id = getId('Rating');
+    this._labelId = getId('RatingLabel');
+  }
+
+  public componentWillReceiveProps(nextProps: IRatingProps) {
+    if (typeof nextProps.rating !== 'undefined' && nextProps.rating !== this.state.rating) {
+      this.setState({
+        rating: this._getClampedRating(nextProps.rating)
+      } as IRatingState);
+    }
+  }
+
+  public render() {
+    let stars: JSX.Element[] = [];
+    for (let i = this.props.min; i <= this.props.max; ++i) {
+      stars.push(this._renderStar(i));
+    }
+
+    return <div className={ css('ms-Rating', this.props.className, {
+      'ms-Rating--large': this.props.size === RatingSize.Large
+    }) } role='application'>
```

Comment:
I haven't tested it with JAWS but NVDA works fine without additional role `application`. This has two side effects: 
1. Screen reader announces _application_ everytime such control is focused/re-focused which feels like a noise.
2. If I have 10 of these controls in my form, it creates 10 additional application landmarks on the page.

We can choose to keep it this way until we have confirmed JAWS compatibility. Thanks,